src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -815,26 +907,23 @@ def ert_templates(self):
         return self._templates
 
     def __eq__(self, other):
-        # compare each config separatelly
-        config_eqs = (
-            (self.substitution_list == other.substitution_list),
-            (self.site_config == other.site_config),
-            (self.random_seed == other.random_seed),
-            (self.num_cpu_from_config == other.num_cpu_from_config),
-            (self.num_cpu_from_data_file == other.num_cpu_from_data_file),
-            (self.analysis_config == other.analysis_config),
-            (self.ert_workflow_list == other.ert_workflow_list),
-            (self.ert_templates == other.ert_templates),
-            (self.ensemble_config == other.ensemble_config),
-            (self.model_config == other.model_config),
-            (self.queue_config == other.queue_config),
+        return (
+            isinstance(other, ResConfig) and other is not None,
+            self.substitution_list == other.substitution_list
+            and self.license_root_path == other.license_root_path
+            and self.job_list == other.job_list
+            and self.env_vars == other.env_vars
+            and self.random_seed == other.random_seed
+            and self.num_cpu_from_config == other.num_cpu_from_config
+            and self.num_cpu_from_data_file == other.num_cpu_from_data_file
+            and self.analysis_config == other.analysis_config
+            and self.ert_workflow_list == other.ert_workflow_list
+            and self.ert_templates == other.ert_templates
+            and self.ensemble_config == other.ensemble_config
+            and self.model_config == other.model_config
+            and self.queue_config == other.queue_config,
         )
```

Comment:
this looks like it returns a tuple of booleans, no? should the comma in line 911 be replaced with an `and`?

also: any reason for changing from the long tuple and call to `.all` to the conjunction?

Reply:
Less ifs and buts, easier to read, imho.
But, yeah, it helps if it is correct as well :)